Sources/SQLKit/SQLDialect.swift

```diff
@@ -8,6 +8,8 @@ public protocol SQLDialect {
     var literalDefault: SQLExpression { get }
     var supportsIfExists: Bool { get }
     var supportsAutoIncrement: Bool { get }
+    var supportsAutoIncrementUsingDefaultFunction: Bool { get }
+    var autoIncrementFunction: SQLExpression { get }
```

Comment:
These two properties could be combined by having `autoIncrementFunction` be optional (`SQLExpression?`).

Reply:
Ah, yes, that’s kind of obvious. Will fix.